frontend-student/src/components/allcourses/CourseNew.jsx

```
@@ -0,0 +1,92 @@
+              <h2 className="card-title">{course.title}</h2>
+              <div className="flex justify-between items-center mt-4">
+                <div>
+                  <p className="text-primary font-bold">৳ {course.price}</p>
+                  {course.rating === null ? (
+                    <p className="text-sm text-gray-500">⭐ No Ratings</p>
+                  ) : (
+                    <p className="text-sm text-gray-500">⭐ {course.rating} / 5</p>
+                  )}
+                </div>
+                <Link to={`/courses/${course.id}`}>
+                  <button className="btn btn-primary btn-sm">View Details</button>
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)